src/bin/ch-remote.rs

```diff
@@ -382,111 +496,240 @@ fn rest_api_do_command(toplevel: &TopLevel, socket: &mut UnixStream) -> ApiResul
             )
             .map_err(Error::HttpApiClient)
         }
-        SubCommandEnum::Create(ref config) => {
-            let data = create_data(&config.vm_config)?;
+        Some("create") => {
+            let data = create_data(
+                matches
+                    .subcommand_matches("create")
+                    .unwrap()
+                    .get_one::<String>("path")
+                    .unwrap(),
+            )?;
             simple_api_command(socket, "PUT", "create", Some(&data)).map_err(Error::HttpApiClient)
         }
-        SubCommandEnum::Version(_) => {
-            // Already handled outside of this function
-            panic!()
-        }
+        Some(c) => simple_api_command(socket, "PUT", c, None).map_err(Error::HttpApiClient),
```

Comment:
I think this is no longer needed as you already handled the plain command explicitly (say "boot/delete/info", etc).

Reply:
ok